flask-redis/README.md

```diff
@@ -0,0 +1,72 @@
+## Compose sample application
+
+### Python/Flask application using a Redis database
+
+Project structure:
+
+```
+.
+├── Dockerfile
+├── README.md
+├── app.py
+├── docker-compose.yml
+└── requirements.txt
+```
+
+[_docker-compose.yml_](docker-compose.yml)
+
+```
+services:
+   redis: 
+     image: redislabs/redismod
+     ports:
+       - '6379:6379' 
+   web:
+        build: .
+        ports:
+            - "5000:5000"
+        volumes:
+            - .:/code
+        depends_on:
+            - redis
+```
+
+## Deploy with docker-compose
+
+```
+$ docker-compose up -d
```

Comment:
My previous suggestion was to use the `docker compose up -d` output and not the one from `docker compose up`, why did you completely removed logs?

You can commit the following suggestion which corresponds to what we need in the documentation

```suggestion
$ docker-compose up -d
[+] Running 24/24
 ⠿ redis Pulled   
 ...                                                                                                                                                                                                                                                                                                                                                                                                             
   ⠿ 565225d89260 Pull complete                                                                                                                                                                                                      
[+] Building 12.7s (10/10) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                                                  ...
[+] Running 3/3
 ⠿ Network flask-redis_default    Created                                                                                                                                                                                             
 ⠿ Container flask-redis-redis-1  Started                                                                                                                                                                                             
 ⠿ Container flask-redis-web-1    Started
```